stable/oauth2-proxy/templates/deployment.yaml

```diff
@@ -66,6 +66,9 @@ spec:
           - --htpasswd-file=/etc/oauth2_proxy/htpasswd/users.txt
         {{- end }}
         env:
+        {{- if .Values.extraEnv }}
```

Comment:
There is already a `extraEnv` below, what is the difference?

Reply:
Huh. Looks like #16343 did something completely identical and was merged recently. Closing this.